internal/function/blame.go

```diff
@@ -32,7 +36,11 @@ func (g *BlameGenerator) loadNewFile() error {
 
 	result, err := git.Blame(g.commit, g.curFile.Name)
 	if err != nil {
-		return err
+		msg := fmt.Sprintf("Error in BLAME for file %s: %s",
```

Comment:
of one of the parameters is in a newline, then better put each in a new line:

```go
msg := fmt.Sprintf(
    "Error in BLAME for file %s: %s",
    g.curFile.Name,
    err.Error(),
)

Reply:
Changed.